packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx

```diff
@@ -29,9 +33,12 @@ export const AssetCoinLogo = ({ symbol, assetsFiatBalances, index }: CoinLogoPro
     return (
         <LogoWrapper>
             <Tooltip
-                content={`${(assetPercentage ?? 0).toFixed(2)} %`}
+                content={localizePercentage({
+                    value: assetPercentage ?? 0,
+                    locale,
+                    numDecimals: 2,
+                })}
```

Comment:
nit: how about also chageing it in a couple of other places, [here](https://github.com/trezor/trezor-suite/blob/a513f54f8c10fb8d008170397a7cd647ac2ff125/packages/suite/src/components/suite/Ticker/TrendTicker.tsx#L56) for example?

Reply:
:+1: